app/client/stylesheets/_login.scss

```diff
@@ -9,10 +9,28 @@
 #login {
   height: 100%;
   width: 100%;
-  background-color: $login-bg-color;
-  background: $login-bg;
-  background-size: cover;
-  overflow: hidden;
+
+  > img.background {
+    height: 100%;
+    width: 101%;
```

Comment:
Can this be 100%?

Reply:
no its actually a touch off center so this centers it. i know its driving my ocd wild too